README.md

```diff
@@ -55,9 +55,13 @@ Carthage builds your dependencies and provides you with binary frameworks, but y
 1. On your application targets’ _Build Phases_ settings tab, click the _+_ icon and choose _New Run Script Phase_. Create a Run Script in which you specify your shell (ex: `/bin/sh`), add the following contents to the script area below the shell:
 
     ```sh
-    /usr/local/bin/carthage copy-frameworks
```

Comment:
@dimazen  let's but this as *experimental* and leave the documentation for both

I'm suggesting this in case people start depending on some behavior we have finalized yet.

Reply:
Update documentation to match latest master. Appended `auto` section below. Also reordered auto section for `If you're building for iOS, tvOS, or watchOS` section to be lower than manual as well as prepend it with the "Experimental" prefix.